src/extra/components/ppu.py

```diff
@@ -59,9 +59,17 @@ def _find_ppu(run: DataCollection, device: str = None):
 
 
 class PPU:
-    """Interface to a PPU (Pulse Picker Unit).
+    """Interface to a Pulse Picker Unit (PPU).
 
-    Despite its name, the PPU picks trains, not pulses.
+    Despite its name, the PPU selects a bunch train from within the 10Hz
+    structure and block the remainder of the beam.
+
+    Technical description:
+        A motor-driven absorber rotor is rotated into the beam axis in order to
+        block the XFEL beam when triggered. The rotor is contained within a UHV
+        chamber. In terms of temporal structure, the beam pipe is blocked by an
+        absorbing rotor for up to 9/10ths of a second or vice versa,
+        synchronized to the facility clock/trigger.
```

Comment:
Very useful information for the docs, I only wonder (naive question) how the 9/10 blocking scheme, i.e. one train per second picked, fits to an overall run where 1 / 309 trains are selected - like run 3379, 50 for instance, shouln't it then be 30 trains with non-consecutive IDs?